docs/contents/user-docs/cleanup.md

```diff
@@ -0,0 +1,60 @@
+# Cleanup 
+
+Few custom resource definitions (CRDs) are common across services. If you have multiple controllers installed, you should not delete the common CRDs unless you are uninstalling all of the controllers.
+
+## Uninstall ACK service controllers and CRDs
+
+Use the following commands to uninstall ACK service controllers and delete CRDs:
+```bash
+export SERVICE=sagemaker
```

Comment:
nit: why sagemaker here? all the rest of the docs referred to s3. We should keep the consistency.